doc/classes/String.xml

```diff
@@ -670,8 +670,9 @@
 			</argument>
 			<description>
 				Splits the string by a [code]divisor[/code] string and returns an array of the substrings, starting from right.
-				[b]Example:[/b] [code]"One,Two,Three"[/code] will return [code]["One","Two","Three"][/code] if split by [code]","[/code].
-				If [code]maxsplit[/code] is specified, then it is number of splits to do, default is 0 which splits all the items.
+				[b]Example:[/b] [code]"One,Two,Three,Four"[/code] will return [code]["Three","four"][/code] if split by [code]","[/code] with maxsplit of [code]"2"[/code].
```

Comment:
"four" should be capitalized in the second `[code]` to match the original string. The example should also come after the line that explains what maxsplit is (and likely after the last line about the return value ordering too).